templates/main-page.html

```diff
@@ -1,24 +1,47 @@
-{% extends "auth.html" %}
-
-{% block body %}
+{% extends "auth.html" %} {% block body %}
 <div class="px-8 py-12 relative text-xl">
-    <a class="github-fork-ribbon left-top z-10" href="https://github.com/felienne/hedy" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
+    <a
+        class="github-fork-ribbon left-top z-10"
+        href="https://github.com/felienne/hedy"
+        data-ribbon="Fork me on GitHub"
+        title="Fork me on GitHub"
+        >Fork me on GitHub</a
```

Comment:
This forced line-breaking makes the code harder to read and doesn't change anything. Is it possible to reverse this?

Reply:
Ah, that's the automatic formatting tool in VSCode. Yes, I can revert that.